### ISSUE TITLE
New parser: `chage -l`

### ISSUE THREAD

**Issue by kkirsche** [kellyjonbrazil/jc#241]
This issue is a request to add support for the `chage` command.

https://linux.die.net/man/1/chage

Specifically, the `chage --list` 

Example output:

```
Last password change					: Mar 02, 2022
Password expires					: never
Password inactive					: never
Account expires						: never
Minimum number of days between password change		: 0
Maximum number of days between password change		: 99999
Number of days of warning before password expires	: 7
```

## Motivation

The `chage` command provides valuable information in automation workflows when checking for security compliance if password expiration or account expiration are required.

Thanks for your time and for the great tool

**Comment by kellyjonbrazil** [comment]
Thank you for the parser suggestion! I think this would be a good fit and looks pretty simple to parse. I can probably get this out in the next release.

**Comment by kellyjonbrazil** [comment]
I have a working version to test in the `dev` branch. You can copy the [`chage.py`](https://github.com/kellyjonbrazil/jc/blob/dev/docs/parsers/chage.md) file to your [plugin directory](https://github.com/kellyjonbrazil/jc#custom-parsers) to test.

Let me know if you run into any issues. Thanks!

**Comment by kkirsche** [comment]
I just tested that via the `jc` command line, and it worked perfectly.

Once I did that, I put the following test script together, which also worked perfectly.

```python
#!/usr/bin/env python3

from getpass import getuser
from shlex import split
from subprocess import PIPE, run

from jc import parse


def main() -> None:
    current_user = getuser()
    cmd = split(f"chage -l {current_user}")
    chage_result = run(cmd, check=True, stdout=PIPE)
    parsed_result = parse("chage", chage_result.stdout.decode("utf-8"))
    print(parsed_result)


if __name__ == "__main__":
    main()
```

Result:

```javascript
{'password_last_changed': 'May 11, 2022', 'password_expires': 'never', 'password_inactive': 'never', 'account_expires': 'never', 'min_days_between_password_change': 0, 'max_days_between_password_change': 99999, 'warning_days_before_password_expires': 7}
```

which matches the data from `chage`.

Thank you so much for your help and for such a fast turnaround!

**Comment by kellyjonbrazil** [comment]
Awesome - thanks for testing! I'll get this into the next release in the next couple of weeks.

**Comment by kellyjonbrazil** [comment]
This parser is now available via pip on version 1.19.0

**Comment by kkirsche** [comment]
Thank you!
